cal/api/views.py

```diff
@@ -242,4 +242,4 @@ class TagDetailEventTimeSeries(APIView):
 
     def get(self, request, *args, **kw):
         tag = Tag.objects.get(user=self.request.user, id=self.kwargs['pk'])
-        return Response(tag.get_time_series(self.request.query_params.get('timezone')))
+        return Response(tag.get_time_series(self.request.query_params.get('timezone'), time_step=self.kwargs['time_step']))
```

Comment:
Wait can't you just do .lower() here to convert to lower case to send to the function?